web/src/containers/BudgetSummary.js

```diff
@@ -61,7 +61,7 @@ class DataRow extends Component {
             {hasData && (
               <button
                 type="button"
-                className="right btn px-tiny py0"
+                className="btn right px-tiny py0"
                 onClick={this.toggleDetails}
               >
```

Comment:
I'm guessing this is a special case button because it's not `primary` or `outline` and it's `tiny`.  Well, special cases will happen. 😛